portfolio/src/main/webapp/script.js

```diff
@@ -13,17 +13,35 @@
 // limitations under the License.
 
 /**
- * Adds a random Star Wars Quote to the page.
+ * Adds a random quote to the page.
  */
-function addRandomStarWarsQuote() {
+function addRandomQuote() {
   const quotes =
-      ['Help me, Obi-Wan Kenobi. You are my only hope. -Lelia Organa', 'The force will be with you. Always. -Obi Wan Kenobi', 
```

Comment:
Awww I liked the star wars quotes :(

Reply:
I did too. I think I will add them back along with the other quotes I added.